### PR TITLE
[react-places-autocomplete] add session token to searchOptions

### PR DESCRIPTION
**Pull request by marknelissen**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://developers.google.com/maps/documentation/javascript/reference/places-autocomplete-service#AutocompletionRequest>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/react-places-autocomplete/index.d.ts [hunk 1]
```diff
@@ -4,8 +4,9 @@
 //                 Andrew Makarov <https://github.com/r3nya>
 //                 Nokky Goren <https://github.com/ApeNox>
 //                 Aziz Khambati <https://github.com/azizhk>
+//                 Mark Nelissen <https://github.com/marknelissen>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.7
```
Comment (r3nya) [inline types/react-places-autocomplete/index.d.ts]: Hi! Please revert this line. This is not necessary for this PR.

**Review by r3nya — CHANGES_REQUESTED** [review]

File: types/react-places-autocomplete/index.d.ts (hunk 1 above)
Comment (marknelissen) [reply types/react-places-autocomplete/index.d.ts]: Actually, it is. Without the Typescript version update, the CI build fails. It's the error of the second commit. The error is in the googlemaps typings dependency.
[Failed Travis build](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/630561537?utm_source=github_status&utm_medium=notification)

afbeelding

Comment (r3nya) [reply types/react-places-autocomplete/index.d.ts]: Oh, I see.

I've checked types for googlemaps -> https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/googlemaps/index.d.ts#L19

It use typescript 3.0 version. Can you bump TS version to 3.0 please? :)

Comment (r3nya) [inline types/react-places-autocomplete/index.d.ts]:
```suggestion
// TypeScript Version: 3.0
```

**Review by r3nya — APPROVED** [review]

File: types/react-places-autocomplete/index.d.ts [hunk 2]
```diff
@@ -38,6 +39,7 @@ export interface PropTypes {
         location?: google.maps.LatLng | google.maps.LatLngLiteral;
         offset?: number | string;
         radius?: number | string;
+        sessionToken?: any;
```
Comment (azizhk) [inline types/react-places-autocomplete/index.d.ts]: Better to type using:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a8fb783542bd3e38682182ef8fd9225215b2dc19/types/googlemaps/index.d.ts#L3608

Comment (marknelissen) [reply types/react-places-autocomplete/index.d.ts]: According to the documentation, it is not strictly required to use AutocompleteSessionToken. It is just there as a way to make life easier, but anything unique (like a self generated UUID) will suffice as token.

File: types/react-places-autocomplete/index.d.ts (hunk 1 above)
Comment (marknelissen) [reply types/react-places-autocomplete/index.d.ts]: Done

**Pull request merged**